COinS.js

```diff
@@ -302,34 +303,32 @@ var testCases = [
 	},
 	{
 		"type": "web",
-		"url": "http://www.hubmed.org/display.cgi?uids=21665052",
+		"url": "https://www.worldcat.org/title/genealogy-of-the-royal-family-of-great-britain-shewing-the-descent-of-queen-victoria-through-five-of-the-saxon-kings-who-were-crowned-at-kingston-upon-thames/oclc/559794003&referer=brief_results",
 		"items": [
 			{
-				"itemType": "journalArticle",
+				"itemType": "book",
+				"title": "The Genealogy of the Royal Family of Great Britain, shewing the descent of ... Queen Victoria, through five of the Saxon Kings who were crowned ... at Kingston-upon-Thames.",
 				"creators": [
 					{
-						"creatorType": "author",
-						"firstName": "Hui-Wen Vivian",
-						"lastName": "Tang"
+						"lastName": "Royal Families (England)",
+						"isInstitution": true
```

Comment:
Looks like this might just be an bug in openurl.js (which has probably been present for 15 years), but `isInstitution` isn't a valid property here. It should be `fieldMode: 1`.